src/MLJInterface.jl

```diff
@@ -216,7 +216,7 @@ function fit(mlj_model::MODELS, verbosity::Int, X, y, w=AbstractFloat[])
     X = MLJModelInterface.matrix(X)
     # The FFI wrapper wants Float32 for these
     w = Float32.(w)
-    train_metrics = LightGBM.fit!(model, X, y_lgbm; verbosity=verbosity, weights=w)
+    train_metrics = LightGBM.fit!(model, X, y_lgbm[:]; verbosity=verbosity, weights=w)
```

Comment:
I know why this fixes it but it isn't necessarily obvious to the casual reader: perhaps put a comment explaining that slicing notation creates copies and LGBM needs contiguous in memory data?

Reply:
done :)